JASP-Desktop/data/datasetpackage.h

```diff
@@ -107,7 +107,7 @@ class DataSetPackage : public QAbstractItemModel //Not QAbstractTableModel becau
 		const	Json::Value		&	analysesData()						const	{ return _analysesData;							 }
 		const	std::string		&	warningMessage()					const	{ return _warningMessage;						  }
 		const	Version			&	archiveVersion()					const	{ return _archiveVersion;						   }
-		const	emptyValsType	&	emptyValuesMap()					const	{ return _emptyValuesMap;   						}
+				emptyValsType	&	emptyValuesMap()							{ return _emptyValuesMap;   						}
```

Comment:
@boutinb this kind of breaks the whole "datasetpackage does everything"-paradigm, doesn't it?

Because this way the emptyValuesMap could be changed at any time anywhere right? And nothing would be triggered, not packageModified, not modelChanged, nothing...

Reply:
OK, I have set back the const, and change the emptyValuesMap in DataSetPackage, not in DataSet anymore.